build/cloudbuild.yaml

```diff
@@ -0,0 +1,17 @@
+# Build the module.
+steps:
+- name: gcr.io/cloud-builders/go:debian
+  entrypoint: bash
+  args:
+  - -c
+  - |
+    set -ex
+    IFS=- read str1 rev <<< "${_GIT_TAG}"
+    make build REV=${rev}
+    cp bin/csi-proxy.exe bin/csi-proxy${_GIT_TAG}.exe
```

Comment:
We may want to consider enhancing this later with the logic in https://github.com/kubernetes-csi/csi-release-tools/pull/87/files#diff-841c5d7c9fb7c0ba0fb2f452298ed526R147 to support "canary" images

Reply:
got it.